### ISSUE TITLE
Warning message from pynndescent

### ISSUE THREAD

**Issue by ivan-marroquin** [pavlin-policar/openTSNE#192]
Hi,

Many thanks for such good package!

I am using pynndescent 0.5.2 and opentsne 0.6.0, when I use these commands:

affinities_multiscale= openTSNE.affinity.Multiscale(input_data, perplexities= [120, 60], method= 
                                   'pynndescent', metric= 'minkowski', metric_params= {'p': 0.3},
                                   n_jobs= cpu_count, random_state= 1969)
    
embedding= openTSNE.TSNE(n_components= intrinsic_dim, perplexity= 120, learning_rate= 
                      'auto', early_exaggeration_iter= 100, early_exaggeration= 22, n_iter= 350,
                      theta= 0.5, initialization= 'random', neighbors= 'pynndescent', metric= 
                      'minkowski', metric_params= {'p': 0.3}, initial_momentum= 0.5, 
                      negative_gradient_method= 'bh', n_jobs= cpu_count, random_state= 1969)
    
projected_data= embedding.fit(input_data, affinities= affinities_multiscale)

I get the following warning message:
c:\temp\python\python3.6.5\lib\site-packages\openTSNE\nearest_neighbors.py:452: UserWarning: `pynndescent` has recently changed which distance metrics are supported, and `openTSNE.nearest_neighbors` has not been updated. Please notify the developers of this change.
  "`pynndescent` has recently changed which distance metrics are supported, "

Ivan

**Comment by pavlin-policar** [comment]
Thanks for letting me know! I'll get on it ASAP.
